data/INRA/concepts.ttl

```diff
@@ -0,0 +1,18 @@
+bdg:Variety a rdf:Class, skos:Concept;
```

Comment:
Lets start keeping the concepts and the coded lists in the same file - it will be easier to maintain

Reply:
I will move them from other dataset file to concept file.

Reply:
Start to work on it fdab11ad1dad11a91dd5a6e306506214ebe09852